frontend/public/components/cluster-settings/github-idp-form.tsx

```diff
@@ -221,7 +221,11 @@ class AddGitHubPageWithTranslation extends PromiseComponent<
           <div className="co-form-section__separator" />
           <h3>{t('public~Organizations')}</h3>
           <p className="co-help-text">
-            <Trans i18nKey="teams-help" ns="public">
+            <Trans
+              t={t}
+              ns="public"
+              i18nKey="Optionally list organizations. If specified, only GitHub users that are members of at least one of the listed organizations will be allowed to log in. Cannot be used in combination with <strong>teams</strong>."
```

Comment:
Why this needed to stay? Whats the special kay that its not able to handle?

Reply:
The parser wasn't correctly generating the <strong> key. This does an end run around that issue since we're not using the parser to generate the keys in this case. i18next can handle strong just fine, but apparently sometimes it makes the parser choke. I used the react-i18next default Codepen to check to make sure i18next was ok and narrow down the issue to the parser.

Reply:
And what about other appearances of `<strong>` tag?, eg. `frontend/public/components/image-stream.tsx` ? is that safe to use there